person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  # attribute accessor for getters and setters
+  attr_accessor :id, :name, :age
+
+  # constructor
+  def initialize(age, name = 'unknown', parent_permission: true) # rubocop:disable Lint/UnusedMethodArgument
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+  end
```

Comment:
- [ ] Kindly ensure you initialize the parent permission correctly, parent permission should be parent_permission = true since you are assigned true to the parent permission, also you forgot to assign the passed parameter to a variable inside the initialized method like you did for name and age.